src/mixins/filing-template-mixin.ts

```diff
@@ -432,11 +432,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           identifier: this.getBusinessId
         },
         contactPoint: {
-          email: this.getBusinessContact.email,
-          phone: this.getBusinessContact.phone,
-          ...this.getBusinessContact.extension
-            ? { extension: +this.getBusinessContact.extension }
-            : {}
+          email: this.getBusinessContact.email || undefined, // don't include if empty
+          phone: this.getBusinessContact.phone || undefined, // don't include if empty
+          extension: +this.getBusinessContact.extension || undefined // don't include if empty
```

Comment:
@argush3 What do you think?

Reply:
I think this looks good other than the email being empty.  it will fail schema validation if it's not included.

https://github.com/bcgov/business-schemas/blob/e32e6dc561abebdc3034af48993da12f882dfb80/src/registry_schemas/schemas/contact_point.json#L4-L6

Reply:
Thanks for the second set of eyes. I totally missed that the email was required in the schema.

I updated this (and testing ASAP). Please have another look.

Reply:
Empty string for contact point email did not work for one of my test conv businesses, so I added an actual string default. It's not ideal but we can discuss and change it later.

Reply:
^^ This is working for now.